handout/features/spread_and_rest.md

```diff
@@ -1,14 +1,14 @@
-# Operators Spread and Rest
+# Spread Syntax (Spread Element) and Rest parameters
 
-A Spread operator allows in-place expansion of an expression for the following cases:
+A Spread syntax allows in-place expansion of an expression for the following cases:
```

Comment:
Also, it's weird that at the end they are using the term "Rest operator" when in another section they are referring to "Rest parameters". I don't know what to believe anymore...